examples/tcp-udp/README.md

```diff
@@ -0,0 +1,93 @@
+# TCP/UDP Load Balancing Example
+
+In this example we deploy the NGINX Plus OpenShift Router, a DNS server and then configure both TCP and UDP load balancing for the DNS server using routes.
+
+## Prerequisites
+
+* We use `dig` for testing. Make sure it is installed on your machine.
```

Comment:
Add an additional prerequisite:

Make ensure to open port 5353 in the firewall for both TCP and UDP traffic on the node where the Router is running: 
```
$ sudo iptables -I OS_FIREWALL_ALLOW -p tcp --dport 5353 -j ACCEPT
$ sudo iptables -I OS_FIREWALL_ALLOW -p udp --dport 5353 -j ACCEPT
```